src/js/ads.js

```diff
@@ -23,16 +23,16 @@ module.exports = [
   {
     name: 'Critter Mound',
     url: 'http://yoyz.com/critter/',
-    text: 'Breed new critters, breed stronger critters, destroy your enemies.'
+    text: 'Breed new critters. Breed stronger critters. Destroy your enemies.'
   },
   {
     name: 'Progress Quest',
     url: 'http://progressquest.com/play/',
-    text: 'Progress Quest is an antic and fantastical computer role-playing game, well worth the time you\'ll spend playing it.'
+    text: 'Progress Quest is an antic and fantastical computer role-playing game. It's well worth the time you\'ll spend playing it.'
```

Comment:
Line is too long.<br>Expected '}' to match '{' from line 28 and instead saw 's'.<br>Expected ']' to match '[' from line 2 and instead saw 'well'.<br>Missing semicolon.<br>Expected an assignment or function call and instead saw an expression.<br>Unexpected '\'.<br>'worth' is not defined.<br>'the' is not defined.<br>'time' is not defined.<br>'you' is not defined.